vlayer/ethereum/circuits/lib/src/slot_test.nr

```diff
@@ -36,6 +36,23 @@ mod mapping {
     }
 }
 
+mod mapping_by_uint {
+    use crate::slot::mapping_by_uint;
+
+    #[test]
+    fn uint_mapping() {
+        let slot = [
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10
+        ];
+        let uint = 0;
+        assert_eq(
+            [
+            0x13, 0xda, 0x86, 0x00, 0x8b, 0xa1, 0xc6, 0x92, 0x2d, 0xae, 0xe3, 0xe0, 0x7d, 0xb9, 0x53, 0x05, 0xef, 0x49, 0xeb, 0xce, 0xd9, 0xf5, 0x46, 0x7a, 0x0b, 0x86, 0x13, 0xfc, 0xc6, 0xb3, 0x43, 0xe3
```

Comment:
Let's name this value. expected_slot? 9th_owner_slot?

Reply:
Fixed, before it was looking like magic number indeed